pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -859,6 +859,32 @@ func getSRIOVPCIAddresses(ifaces []v1.Interface) map[string][]string {
 	return networkToAddressesMap
 }
 
+// This function parses the NVIDIA-PASSTHROUGH-DEVICES variable that is set by
+// device plugin. It lists PCI IDs for devices allocated to the pod. The format
+// is as follows:
+//
+// "": for no allocated devices
+// "0000:81:11.1,": for a single device
+// "0000:81:11.1,0000:81:11.2[,...]": for multiple devices
```

Comment:
I think we should generalize the example too. It only addresses PCI addresses but the are mdevs are well that should look like `/sys/devices/pci0000:00/0000:00:02.0/4b20d080-1b54-4048-85b3-a6a62d165c01`

Also, perhaps, the name of the function should be  getGpuAddresses, to not make it specific for pci

Reply:
Updated the comment and  function name.